components/DatasetDetails/DatasetAboutInfo.vue

```diff
@@ -121,11 +121,14 @@ export default {
     getRecordsUrl: function() {
       return `${process.env.discover_api_host}/search/records?datasetId=${this.$route.params.datasetId}`
     },
-    associatedPublications: function() {
-      return this.externalPublications.length > 0
-        ? 'Associated Publications'
-        : 'Associated Publication'
-    }
+
+     primaryPublication: function() {
+        var valObj = this.externalPublications.filter(function(elem){
+            if(elem.Name == "IsDescribedBy") return elem;
+        });
+        if(valObj.length > 0)
+            return valObj[0]
+     }
```

Comment:
A computed property should always return a value, even if that is `undefined` or `null`.
```suggestion
     primaryPublication: function() {
      const valObj = this.externalPublications.filter(function(elem) {
        return elem.Name == 'IsDescribedBy'
      })
      return valObj.length > 0 ? valObj[0] : null
    }
```

Reply:
👍🏻